### ISSUE TITLE
[FEATURES REQUEST] - Add options

### ISSUE THREAD

**Issue by J316** [gort818/ddgtk#2]
- ~~Add `status=progress` option by default~~
- Add a drop-down menu to change `bs` option
- Add the possibility to use an external usb device as if and specify an output file for backing up units

**Comment by gort818** [comment]
@J316 status=progress is already set by default. Would you like the option to remove the option? 
I tried with bs=1M but for some reason the status would not show.

But I guess I could add a checkbox for the options with an expander menu. What do yo think?

**Comment by J316** [comment]
Yeah, `status=progress` is there by default, apologies for my mistake, there's absolutely no reason for an option to remove it. (I updated the list)
I don't know why changing the block size would cause the status not to show, but it's critical to have the possibility to choose it, because [speed can be increased drastically by tuning that option](http://blog.tdg5.com/tuning-dd-block-size).

**Comment by gort818** [comment]
@J316 Yeah I thought that was strange too, initially I had it set to 1M. I will do some tests later on and let you know of my findings.

**Comment by gort818** [comment]
@j316 I have not forgot about this yet! sorry for the delay! Will try to get on it!

**Comment by J316** [comment]
@gort818 Thanks. I appreciate your commitment

**Comment by gort818** [comment]
@J316 I added some bs options let me know what you think [ 0eb1bae ](https://github.com/gort818/ddgtk/commit/0eb1bae0a698c52d4e0134ba2c9a6f62b85476c3)

**Comment by J316** [comment]
To me it's good because I found 1M works the best, maybe someone else could need others, like 512K or I don't know
